<launch>

 
 <node pkg="tf" type="static_transform_publisher" name="static_broadcaster" args="1 0 0 0 0 0 1 map odom 10" />
  
  <include file="$(find outdoor_bot)/launch/includes/robot_state_publisher_outdoor.launch"/> 
  
   <node pkg="outdoor_bot" type="movement_node" name="movement_node" output= "screen">
  </node>
  
  <node pkg="outdoor_bot" type="robotPose_node" name="robotPose_node" output= "screen">
  </node>
  
    <node pkg="outdoor_bot" type="baseController_node" name="baseController_node" args = "outdoor" output= "screen">
 </node>
 
 <include file="$(find outdoor_bot)/launch/includes/move_base_outdoor.launch"/>
 
 </launch>

 
